.travis.yml

```diff
@@ -73,7 +73,7 @@ matrix:
 
       script:
       - cmake -DCMAKE_BUILD_TYPE=$BUILD_TYPE . && make && make test
-      - if [ "$CONAN" == "true" ] && [ "$TRAVIS_OS_NAME" == "linux" ]; then sudo pip install conan && conan create . flatbuffers/testing -s build_type=$BUILD_TYPE; fi
+      - if [ "$CONAN" == "true" ] && [ "$TRAVIS_OS_NAME" == "linux" ]; then sudo pip install conan && conan create . aardappel/testing -s build_type=$BUILD_TYPE -tf conan/test_package; fi
```

Comment:
Wouldn't it better to use google as the username? This would create some consistency across Google libraries. E.g. I have suggested to use `@google` for the benchmark library as well https://github.com/google/benchmark/pull/647

Reply:
Done!